readme.txt

```diff
@@ -50,6 +50,9 @@ Below the text box is a button that will allow you to simply save all of that in
 
 **Note:** This information contains potentially senstive data. Please be careful with where you post it. Do not post it in the WordPress support forums.
 
+= What external services do we communicate to? =
+We only ping the Content Workflow API to fetch and push data. We do not send any data to any other services. We shall also be pulling your avatar from your account to display in the plugin.
```

Comment:
❓ Does this fit under the third party services stuff? Or does WordPress want this explicitly stated outside of the third-party stuff?

Reply:
I think what you've written probably covers it, didn't realise you had also done it. I'll wait for your to do in then merge and remove my bit